src/helpers/promise.js

```diff
@@ -1,4 +1,5 @@
 /* eslint no-proto:0 */
+/* eslint no-use-before-define:0 */
```

Comment:
I think the more explicit way would be to add `PromiseExt` as a [global variable](http://eslint.org/docs/user-guide/configuring#specifying-globals) for ESLint.